oscillatingCollimatorApp/Db/oscillatingCollimator.db

```diff
@@ -218,243 +280,246 @@ record(ao, "$(P)$(O):ANGLE:SP") {
+    field(INPA, "$(P)$(O):DIST CP MS")
+    field(INPB, "$(P)$(O):DIST:VEL CP MS")
+    field(CALC, "A-2*B")
 }
 
 record(calc, "$(P)$(O):ANGLE") {
     field(DESC, "Actual swept angle")
     field(EGU, "deg")
     field(PREC, "2")
-	field(INPA, "$(P)$(O):DIST:PART CP")
-	field(INPB, "$(P)$(O):RADIUS CP")
-	field(INPC, "$(P)$(O):STEPS CP")
-	field(CALC, "R2D*ATAN(A/(2*B*C))")
+    field(INPA, "$(P)$(O):DIST:PART CP MS")
+    field(INPB, "$(P)$(O):RADIUS CP MS")
+    field(INPC, "$(P)$(O):STEPS CP MS")
+    field(CALC, "R2D*ATAN(A/(2*B*C))")
     info(INTEREST, "HIGH")
-	info(archive, "VAL")
+    info(archive, "VAL")
     field(ASG, "READONLY")
 }
 
 record(calc, "$(P)$(O):CYCLE:TIME:HALF") {
     field(DESC, "Actual half cycle time")
     field(EGU, "s")
     field(PREC, "2")
-	field(INPA, "$(P)$(O):ACC CP")
-	field(INPB, "$(P)$(O):VEL CP")
-	field(INPC, "$(P)$(O):DIST:PART CP")
-	field(CALC, "2*B/A+C/B")
+    field(INPA, "$(P)$(O):ACC CP MS")
+    field(INPB, "$(P)$(O):VEL CP MS")
+    field(INPC, "$(P)$(O):DIST:PART CP MS")
+    field(CALC, "2*B/A+C/B")
 }
 
 record(calc, "$(P)$(O):FREQ:SP:RBV") {
     field(DESC, "Expected operating frequency")
     field(EGU, "Hz")
     field(PREC, "2")
-	field(INPA, "$(P)$(O):CYCLE:TIME:HALF CP")
-	field(CALC, "1/(2*A)")
+    field(INPA, "$(P)$(O):CYCLE:TIME:HALF CP MS")
+    field(CALC, "1/(2*A)")
 }
 
 # The fields set by the calculations
 
 record(ao,"$(P)$(O):VEL:SP") {
-	field(DESC, "Velocity setpoint")
-	field(DTYP, "$(OTYP)")	
-	field(EGU, "stp/s")
-	field(PREC, "2")
-	field(OUT, "@asyn(Galil,0)USER_VAR vel")
-	info(archive, "VAL")
+    field(DESC, "Velocity setpoint")
+    field(DTYP, "$(OTYP)")
+    field(EGU, "stp/s")
+    field(PREC, "2")
+    field(OUT, "@asyn(Galil,0)USER_VAR vel")
+    info(archive, "VAL")
     info(autosaveFields, "VAL")
 }
 
 record(ao,"$(P)$(O):ACC:SP") {
-	field(DESC, "Acceleration setpoint")
-	field(VAL, "49152")
-	field(DTYP, "$(OTYP)")	
-	field(EGU, "stp/s^2")
-	field(PREC, "2")
-	field(OUT, "@asyn(Galil,0)USER_VAR accel")
-	info(archive, "VAL")
+    field(DESC, "Acceleration setpoint")
+    field(VAL, "49152")
+    field(PINI, "YES")
+    field(DTYP, "$(OTYP)")
+    field(EGU, "stp/s^2")
+    field(PREC, "2")
+    field(OUT, "@asyn(Galil,0)USER_VAR accel")
+    info(archive, "VAL")
 }
 
 record(ao,"$(P)$(O):DIST:SP") {
-	field(DESC, "Total swept distance setpoint")
-	field(DTYP, "$(OTYP)")	
-	field(EGU, "stp")
-	field(PREC, "2")
-	field(OUT, "@asyn(Galil,0)USER_VAR dist")
-	info(archive, "VAL")
+    field(DESC, "Total swept distance setpoint")
+    field(DTYP, "$(OTYP)")
+    field(EGU, "stp")
+    field(PREC, "2")
+    field(OUT, "@asyn(Galil,0)USER_VAR dist")
+    info(archive, "VAL")
     info(autosaveFields, "VAL")
 }
 
 # Recover the variables set in the Galil
 
 record(ai,"$(P)$(O):VEL") {
-	field(DESC,"Velocity")
-	field(SCAN,"1 second")
-	field(DTYP,"$(OTYP)")
-	field(INP, "@asyn(Galil,0)USER_VAR vel")
+    field(DESC,"Velocity")
+    field(SCAN,".1 second")
```

Comment:
vel, acc and dist don't actually change during oscillation. Why do they need a higher update rate?

Reply:
I have slowed them back down